code/datums/ruins/space.dm

```diff
@@ -314,3 +314,21 @@
 	suffix = "reticencefacility.dmm"
 	name = "Abandoned Mime Facility"
 	description = "This facility seems to have been quickly abandoned. It appears to be related to Mimes."
+
+/datum/map_template/ruin/space/spiderclanstation
```

Comment:
could you pls pls pls modularize this?
just put these 3 templates into
`voidcrew/code/datums/ruins/space.dm` (you will have to make some of those folders if they don't already exist)
and then just add
`#include "voidcrew\code\datums\ruins\space.dm` into `shiptest.dme` (alphabetically in the voidcrew section)